servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -70,11 +71,11 @@ def update_job_labels(id, body, **kwargs):
         headers=kwargs.get('auth_headers'))
 
     if response.status_code == InternalServerError.code:
-        raise InternalServerError(response.json().get('message'))
+        raise InternalServerError(_get_response_massage(response))
     elif response.status_code == BadRequest.code:
-        raise BadRequest(response.json().get('message'))
+        raise BadRequest(_get_response_massage(response))
     elif response.status_code == NotFound.code:
-        raise NotFound(response.json().get('message'))
+        raise NotFound(_get_response_massage(response))
```

Comment:
I'm not 100% sure this would be the right time to get a massage...

Reply:
good point – changed!